docs/integrations/sources/google-search-console.md

```diff
@@ -121,6 +105,52 @@ Parameter `data_state='all'` should not be used with Incremental Sync mode as it
 - [Analytics site report by site](https://developers.google.com/webmaster-tools/search-console-api-original/v3/searchanalytics/query)
 - Analytics report by custom dimensions
 
+## Connector-specific configurations
+
+### Custom reports
+
+Custom reports allow you to query the API with a custom set of dimensions to group results by. Results are grouped in the order that you supply these dimensions. Each custom report should be constructed as a JSON object in the following format:
+
+```json
+{
+   "name": "<report-name>",
+   "dimensions": ["<dimension-name>", "<dimension-name>", ...]
+}
+```
+
+The available dimensions are:
+
+- `country`
+- `date`
+- `device`
+- `page`
+- `query`
+- `searchAppearance`
+
+For example, to query the API for a report that groups results by country, then by date, you could enter the following custom report:
+
+```json
+[
+   {
+      "name": "country_date",
+      "dimensions": ["country", "date"]
+   }
+]
+```
+
+You can use the [Google APIS Explorer](https://developers.google.com/webmaster-tools/v1/searchanalytics/query) to build and test the reports you want to use.
+
+### Data State
```

Comment:
should this option and section be called data freshness instead?

Reply:
great addition on this section btw, nice work!

Reply:
Thanks! Yeah I think data freshness makes sense, I've made the changes here and added the new title to the spec PR